docs/content/en/latest/architecture/supported_frameworks/_index.md

```diff
@@ -6,7 +6,9 @@ weight: 28
 no_list: true
 ---
 
-As of now, there is direct support only for [React](../../quick_start/) and [Angular 9+](./angular/). There are no near-future plans to introduce guides for any other platforms or frameworks.
+As of now, there is direct support only for [React](../../quick_start/) and [Angular 9+](./angular/) and indirect for [Next.js](./nextjs/)
```

Comment:
As of now, we provide direct support for [React](../../quick_start/) and [Angular 9+](./angular/) and limited support for [Next.js](./nextjs/).